CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 cmake_minimum_required(VERSION 2.8.3)
+add_compile_options(-std=c++11)
```

Comment:
Please rebase the patch against the latest master. This compile option has been added in the meantime already.

Reply:
This comment has not been addressed yet?